### PR TITLE
Db 797 single db

### PR DESCRIPTION
**Pull request by jworcestBAH**
Merges databases into a single DB

### CODE REVIEW

File: tests/baseTestAPI.py [hunk 1]
```diff
@@ -184,10 +181,7 @@ def setUp(self):
     def tearDownClass(cls):
         """Tear down class-level resources."""
         cls.interfaces.close()
-        dropDatabase(cls.interfaces.userDb.dbName)
-        dropDatabase(cls.interfaces.jobDb.dbName)
-        dropDatabase(cls.interfaces.errorDb.dbName)
-        dropDatabase(cls.interfaces.validationDb.dbName)
+        #dropDatabase(cls.interfaces.jobDb.dbName)
```
Comment (bsweger) [inline tests/baseTestAPI.py]: Is a debug leftover?

File: tests/baseTestValidator.py [hunk 2]
```diff
@@ -79,9 +76,7 @@ def setUp(self):
     def tearDownClass(cls):
         """Tear down class-level resources."""
         cls.interfaces.close()
-        dropDatabase(cls.interfaces.jobDb.dbName)
-        dropDatabase(cls.interfaces.errorDb.dbName)
-        dropDatabase(cls.interfaces.validationDb.dbName)
+        #dropDatabase(cls.interfaces.jobDb.dbName)
```
Comment (bsweger) [inline tests/baseTestValidator.py]: debug leftover?

File: dataactcore/migrations/env.py [hunk 3]
```diff
@@ -1,9 +1,13 @@
 from __future__ import with_statement
 from alembic import context
+# Load all DB tables into metadata object
+from dataactcore.models import domainModels
```
Comment (bsweger) [inline dataactcore/migrations/env.py]: do we need these imports? I had this file open in PyCharm, which says that domainModels, errorModels, jobModels, stagingModels, userModel, and validationModels are unused.

**Comment by bsweger** [comment]
@jworcestBAH Done reviewing. 6,872 lines of code deleted--best PR ever!

Just a few comments re: the tearDown classes in the unit tests. 

Lastly, I was unable to run the initialization scripts when I put a custom db name into the config.yml `db_name` field.

File: tests/baseTestAPI.py (hunk 1 above)
Comment (jworcestBAH) [inline tests/baseTestAPI.py]: Uncommented the drop statement.  As all the interfaces point at the same DB now, the drop here can be applied to any one of them.

File: tests/baseTestValidator.py (hunk 2 above)
Comment (jworcestBAH) [inline tests/baseTestValidator.py]: Same as above.

File: dataactcore/migrations/env.py (hunk 3 above)
Comment (jworcestBAH) [inline dataactcore/migrations/env.py]: Without importing these, the Base metadata object does not get the information about the tables and alembic tries to drop them all.  The point of these imports is to load all the tables into the common Base object they all import, which we then get from the next line "from dataactcore.models import baseModel".

**Comment by jworcestBAH** [comment]
Still looking into the custom name issue, I had some difficulties with alembic on this.

**Comment by jworcestBAH** [comment]
Fixed the issue with runMigration for different db names

**Comment by bsweger** [comment]
This all looks good. Feels only right to note that the original design of the db interfaces and models made this merge much less of a big deal than I'd fear. Nice!

Merging!

**Pull request merged**